ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -73,7 +73,7 @@ define([
             //that we have enough values to search
             defaults: function () {
                 return _merge({
-                    cql: "anyText ILIKE ''",
+                    cql: "anyText ILIKE '%'",
```

Comment:
why `%` and not `*` here

Reply:
I saw that's how the CQL strings with `*` queries originally got translated to in requests, so I tried to keep it consistent. Not entirely sure the reasoning behind that, so if it's wrong, I can change it to '*'.